src/kili/services/label_data_parsing/json_response.py

```diff
@@ -66,23 +113,75 @@ def __init__(
                 job_payload=job_response,
             )
 
-    def __getitem__(self, job_name: str) -> "job_response_module.JobPayload":
-        """Returns the JobPayload object corresponding to the job name."""
-        if job_name not in self._json_data:
-            raise JobNotExistingError(job_name)
-        return self._json_data[job_name]
-
-    @property
-    def jobs(self) -> "ParsedJobs":
-        """Returns self."""
-        return self
-
-    def to_dict(self) -> Dict:
+    def to_dict(self) -> Dict[str, Dict]:
         """Returns the parsed json response as a dict."""
         ret = {job_name: job_payload.to_dict() for job_name, job_payload in self._json_data.items()}
         ret = {k: v for k, v in ret.items() if v}  # remove empty json responses
         return ret
 
+
+def _is_video_job(project_info: Project, json_response: Dict) -> bool:
+    """Returns True if the json response is a video job, False otherwise."""
+    if "VIDEO" not in project_info["inputType"]:
+        return False
+
+    if not all(frame_id.isdigit() for frame_id in json_response.keys()):
+        return False
```

Comment:
why not to return an error rather to try to parse it with the ParseOtherJobs class ?
and a more general comment: isn't the test with the inputType enough to know if it is a video response or not ?

Reply:
I see that you raise the error inside ParsedVideoJobs. 
According to me, only the input type should decide if we parse it the video way of the classic way

Reply:
both tests are relevant, maybe the second one could be in another method to separate the concerns

Reply:
No the input type is not enough

The ParsedJobs class can be use several times for the same (root) json response

For example, for a video project, the root json response will be parsed using _ParsedVideoJobs

but if the json response contains children jobs, those job json responses will need to be parsed using _ParsedOtherJobs, despite the input_type being "VIDEO"

Reply:
Ah yeah I get it, thanks